### ISSUE TITLE
Exit status 1 when create client certificate

### ISSUE THREAD

**Issue by ahmadfdr** [d3vilh/openvpn-ui#122]
I have this issue when create client certificate. Actually, i have succeed when create openvpn server for first time in first machine using docker compose method, then i repeat the process on the  different machine but suddenly i have this issue ( i'm sure i have followed the same steps as before and similar to tutorial on this official github page).  
![image](https://github.com/user-attachments/assets/35f50b29-21bf-4d5c-9abf-f4df629ca8c1)

**Comment by HermanRas** [comment]
same problem from clean Docker-Compose using volumes

**Comment by ahmadfdr** [comment]
I have fixed it in unexpected ways wkwk, when u see the docker logs openvpn-ui it shows that exit status code 1 appear because openvpn doesnt find the /easy-rsa/pki direcroty, I have tried to copied manually from repo to target directory that appear on docker logs, but still can't fix the issue. Then, I tried go to Maintenance in Web UI and start to click randomly EasyRSA VARS, Initialize PKI, and Build CA button then suddenly fix the issue:)

**Comment by ahmadfdr** [comment]
> I have fixed it in unexpected ways wkwk, when u see the docker logs openvpn-ui it shows that exit status code 1 appear because openvpn doesnt find the /easy-rsa/pki direcroty, I have tried to copied manually from repo to target directory that appear on docker logs, but still can't fix the issue. Then, I tried go to Maintenance in Web UI and start to click randomly EasyRSA VARS, Initialize PKI, and Build CA button then suddenly fix the issue:)

I assume it's because we only can add or rebuild pki in web ui because it runs on containerization environment, but still don't know clearly wkwk, maybe someone can explain it cleary for knowledge sharing:)

**Comment by HermanRas** [comment]
Sweet, I will try bulding CA and report back.. 

I see the same error in logs, you are correct !
![image](https://github.com/user-attachments/assets/b69b8847-bcb3-4da7-9e62-a48979b15125)

**Comment by ahmadfdr** [comment]
Yes, I tried again and only click the Build CA in web ui and fix the issue immediately

**Comment by HermanRas** [comment]
IT worked for me 2, and I found the problem..

During first Docker volume start the file and folder structure is not initialized correctly,
![image](https://github.com/user-attachments/assets/62eba272-7453-4840-8730-126181fe5cd4)

after running Init All
![image](https://github.com/user-attachments/assets/888f2db4-a941-40e2-976b-f84a9f5e8853)

I was able to Generate the Certs, @ahmadfdr thanks so much !
![image](https://github.com/user-attachments/assets/8fece9df-219c-4569-9520-31ed1863290c)

**Comment by ahmadfdr** [comment]
ah I see, ur welcome @HermanRas  nice to sharing knowledge with you in this community!

**Comment by d3vilh** [comment]
Hi @ahmadfdr Thank you for the reporting.
It is bug un UI backend scripts related to new OpenSSL functionality.
Already fixed in current release as permanent fix will be included into 0.9.5.6.

You could try [this rc image for x86](https://hub.docker.com/repository/docker/d3vilh/openvpn-ui/tags/0.9.5.6-amd64/sha256:6fc021df58889147da5fb50bdb5996b6f1d7b4fff435fb37b355977b35a1800a): `d3vilh/openvpn-ui:0.9.5.6-amd64`

Or [this one for arm64](https://hub.docker.com/repository/docker/d3vilh/openvpn-ui/tags/0.9.5.6-arm64/sha256:d7c2ba5049f8c950e3ddaeebeda45ffbc7651ca5cd38afcef3e8a95696c90cf2): `d3vilh/openvpn-ui:0.9.5.6-arm64`

**Comment by d3vilh** [comment]
I plan to release 0.9.5.6 this year :)

**Comment by ahmadfdr** [comment]
Hahaha congrats man hope still can release in 2024, cz it's about to become 2025 in 5 hours in my time zone🤣
